TTTAttributedLabel.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'TTTAttributedLabel'
-  s.version      = '1.13.4-quizlet'
+  s.version      = 'quizlet-1'
```

Comment:
> we forked somewhere between 1.13.4 and 2.0.0, and now we're in our own world.

Based on https://github.com/quizlet/TTTAttributedLabel/pull/2#issuecomment-637783591, I'm a-ok with this since the original repo hasn't been touched in years so we'll probably never need to sync again. If it had been maintained and was converted to Swift etc then it might have been worth it. 

:shipit: